README.md

```diff
@@ -448,3 +448,54 @@ JSON formatted Zone Report
     }]
 }
 ```
+
+
+### /logicalpath/repl
+Replicates a data object into some resource
+
+**Method**: POST
+
+**Parameters**:
+- all: If selected, updates all stale copies.
+- recursive:  Required to repl a collection. Replicates the whole subtree. Does nothing if used on a data object.
+- num-threads: The number of threads to use for replication.
+- repl-num: Specifies the particular replicate to copy from. Typically not needed.
+- dest-resource: Resource to which to replicate the data object or collection.
+- logical-path: The logical path of the data object or collection to replicate.t
+
+
+**Example CURL command**:
+```
+export TARGET_PATH=/tempZone/home/rods/hello.cpp
+curl -X POST -H "Authorization: ${TOKEN}" 'http://localhost/irods-rest/0.9.1/resource?logical-path=$TARGET_PATH'
+```
+
+### /logicalpath/trim
+
+**Method**: DELETE
+**Parameters**:
+- recursive: Required to trim a collection. Trims the whole subtree. Does nothing if used on a data object.
+- dryrun: If set, no copy will actually be trimmed.
+- age: The minimum age in minutes for a data object to be a candidate for trimmed. If a data object is older, nothing will happen to it.
+- num: The minimum number of copies to leave after trimming. Defaults to 2.
```

Comment:
Consider renaming to `minimum_number_of_replicas`.

Reply:
Maybe even `minimum-number-of-remaining-replicas`? (With '-' instead of '_' to reflect conventions in HTTP?)